test/extensions/http/header_validators/envoy_default/http1_header_validator_test.cc

```diff
@@ -758,30 +761,78 @@ TEST_F(Http1HeaderValidatorTest, InvalidResponseHeaderBeforeSendingDownstream) {
   EXPECT_EQ(result.details(), "uhv.invalid_value_characters");
 }
 
-TEST_F(Http1HeaderValidatorTest, BackslashInPathIsTranslatedToSlash) {
```

Comment:
Test related to backslash are moved to the http_common_validation_test.cc since they are the same for H/1 and H/2.